Assets/PatchKit/UnityEditorExtension/Editor/UI/SafeBuildAndUploadScreen.cs

```diff
@@ -436,6 +445,16 @@ private void BuildAndUpload()
         Assert.IsNull(VersionLabelValidationError);
         Assert.IsNull(VersionChangelogValidationError);
 
+        if (Core.Api.GetAppInfo(new AppSecret(_appSecret)).Removed)
+        {
+            EditorUtility.DisplayDialog(
+                "Removed game",
+                "Your game has been removed and you must connect again with Patchkit.\n\n",

```

Comment:
Change to: `This game does no longer exist on your PatchKit account.`

Reply:
done